packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +139,85 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
+    const uid = uuid();
+    if (this.getAttribute('variant') === 'indeterminate') {
```

Comment:
Maybe we could leave out the variant attribute?

```suggestion
    if (!this.hasAttribute('value')) {
```

Reply:
Yeah, that sounds good.

Reply:
Yeah, that looks good.